content/news/12-05-2025-GalaxyImagingHackathon2025/index.md

```diff
@@ -0,0 +1,51 @@
+
+This wasn’t just about closing issues—it was about shared problem solving and community-driven development. Across those three days, contributors dove into discussions, whiteboarded bold ideas, and tackled technical challenges side by side. By the end, we had made meaningful progress toward expanding Galaxy’s capabilities for image analysis!
+
+## Key Milestones
+A new [Galaxy workflow for Brightfield Microscopy](https://github.com/FAIR-imaging/galaxy-image-community/issues/35), tailored to Masson Trichrome and IHC images, was successfully completed. The [Bellavista visualiser](https://github.com/FAIR-imaging/galaxy-image-community/issues/45) was integrated into Galaxy, bringing spatial transcriptomics data to life. We also released a new [tutorial on parameter tuning and image segmentation evaluation](https://github.com/FAIR-imaging/galaxy-image-community/issues/38), helping users benchmark and optimise segmentation results.
+
+Meanwhile, our tools were tested across diverse file types in a [format compatibility investigation](https://github.com/FAIR-imaging/galaxy-image-community/issues/34), and improvements were made to enhance the user experience—such as restoring the [GitHub link on imaging.usegalaxy.eu](https://github.com/FAIR-imaging/galaxy-image-community/issues/29) and [standardising `.TIFF` vs `.TIF` extensions](https://github.com/FAIR-imaging/galaxy-image-community/issues/43) across tools.
+
+We also took the time to highlight training materials—most notably with the publication of a feature article to promote the BMZ microscopy tutorial, which you can [read here](https://galaxyproject.org/news/2025-04-23-bmz-galaxy/). In parallel, we began designing a general-purpose [“Where to Start” tutorial for image analysis](https://github.com/FAIR-imaging/galaxy-image-community/issues/36) to support newcomers in navigating Galaxy’s imaging landscape.
+
+## Exciting Work Underway
+Development began on integrating [OMERO as a file source](https://github.com/FAIR-imaging/galaxy-image-community/issues/17), as well as [porting Fractal tasks and workflows](https://github.com/FAIR-imaging/galaxy-image-community/issues/31) into the Galaxy ecosystem. We explored extending support for AI inference tools with [TensorFlow compatibility](https://github.com/FAIR-imaging/galaxy-image-community/issues/37), started building a [VPT preprocessing tool for MEROSCOPE images](https://github.com/FAIR-imaging/galaxy-image-community/issues/41), and developed plans for a new tool focused on [gel band quantification using ImageJ](https://github.com/FAIR-imaging/galaxy-image-community/issues/42). A proposal to implement a [more robust TIFF viewer](https://github.com/FAIR-imaging/galaxy-image-community/issues/44) is also underway to better support high-resolution image inspection.
```

Comment:
I'd leave this out because it seems that this was a dead end, unfortunately:
```suggestion
Development began on integrating [OMERO as a file source](https://github.com/FAIR-imaging/galaxy-image-community/issues/17), as well as [porting Fractal tasks and workflows](https://github.com/FAIR-imaging/galaxy-image-community/issues/31) into the Galaxy ecosystem. We explored extending support for AI inference tools with [TensorFlow compatibility](https://github.com/FAIR-imaging/galaxy-image-community/issues/37), started building a [VPT preprocessing tool for MEROSCOPE images](https://github.com/FAIR-imaging/galaxy-image-community/issues/41), and developed plans for a new tool focused on [gel band quantification using ImageJ](https://github.com/FAIR-imaging/galaxy-image-community/issues/42).
```